charon/src/charon-driver.rs

```diff
@@ -210,12 +221,20 @@ fn main() {
     let mut callback = CharonCallbacks::new(options);
     let mut res = callback.run_compiler(compiler_args);
     if let Some(crate_data) = &callback.crate_data {
+        // `crate_data` is `None` on the first call of the driver.
```

Comment:
```suggestion
        // `crate_data` is initialized to `None`, then updated to `Some(...)` after running the compiler
```
I think this is more accurate?

Reply:
That's not what I meant, I'll clarify